### PR TITLE
Implement Rest Services

### PR DESCRIPTION
**Pull request by razvanrosu**

### CODE REVIEW

File: web-crawler/src/main/java/org/iqu/webcrawler/entities/Client.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.iqu.webcrawler.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class Client {
+
```
Comment (andreeabivolaruiquest) [inline web-crawler/src/main/java/org/iqu/webcrawler/entities/Client.java:6]: would be better to initialize variables each on a row.

File: web-crawler/src/main/java/org/iqu/webcrawler/others/PingService.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * Class returns 200 if System runs correctly.
+ * 
+ * @author Razvan Rosu
+ *
+ */
+@Path("/ping")
+public class PingService {
+
+	@HEAD
+	public Response retrievePing() {
```
Comment (andreeabivolaruiquest) [inline web-crawler/src/main/java/org/iqu/webcrawler/others/PingService.java]: could be renamed to ping

File: web-crawler/src/main/java/org/iqu/webcrawler/entities/ErrorMessage.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+/**
+ * ErrorMessage - Class that represents the error message.
+ * 
+ * @author Alex Dragomir
+ *
+ */
+public class ErrorMessage {
+
+	String message;
```
Comment (vladucuvoican) [inline web-crawler/src/main/java/org/iqu/webcrawler/entities/ErrorMessage.java]: please make it a private field if there's no need for default package access

File: web-crawler/src/main/java/org/iqu/webcrawler/event/management/RetrieveEventsService.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+public class RetrieveEventsService {
+
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response retriveEvents(@QueryParam("startDate") String startDate, @QueryParam("endDate") String endDate,
+			@QueryParam("type") String type, @QueryParam("subType") String subType,
+			@QueryParam("sourceId") String sourceId, @QueryParam("author") String author,
+			@QueryParam("location") String location) {
+		
+		String response="";
```
Comment (vladucuvoican) [inline web-crawler/src/main/java/org/iqu/webcrawler/event/management/RetrieveEventsService.java]: Please use the proper spacing when writing code... In eclipse use CTRL+A (Select All) and then CTRL+SHIFT+F (format code)

**Pull request merged**